lib/hammer_cli/output/adapter/abstract.rb

```diff
@@ -7,6 +7,7 @@ def tags
     end
 
     def initialize(context={}, formatters={})
+      context[:verbosity] ||= HammerCLI::V_VERBOSE
```

Comment:
I'm not sure I like setting default verbosity into context here. It should ideally be set only at one place. @mbacovsky what's your take on this?